### ISSUE TITLE
Complex config:  SSL Offload > Cache Static content / pagespeed content / reverse proxy to backend servers with sticky sessions and pass remote ip address. 

### ISSUE THREAD

**Issue by flashtopia** [pagespeed/ngx_pagespeed#478]
I come from a windows background and IIS servers,  My goal is to use nginx pagespeed in front or multiple IIS servers to do elgantly what takes much work on iis.   I am sure that I am not the only IIS windows guy wishing there was a FREE version of pagespeed for iis. 

I finally got NginX custom build installed with mod pagespeed, but am having a very hard time finding a configuration which will accomplish the following goals:
1. SSL Offloading with SPDY support 
2. Cache static content, including content created by pagespeed
3. Follow pagespeed settings to minify / combine scripts and optimize images and other pagespeed goodies. 
4. Get content via reverse proxy (multiple backend servers), and the ability to forward headers, or even use sticky sessions. 

The goal here is to be able to setup a quick Nginx pagespeed server to sit in front of your windows / linux / tomcat / whatever servers.   having a bash script to get the intial config going would be great,  but maybe including a brief wiki article with the above configuration would be extremely useful. 

I know I am not the only one who is stuck using IIS servers or other technology.  And  with linux vms running cheaper than windows, i think using nginx-pagespeed as an all in one solution would be awesome.

To put it simply.  I would love to have Nginx handle all web requests, minify, cache, and offload ssl for a bunch of backend IIS servers.  

THANK YOU FOR ANY TIPS IN ADVANCE.!!!!
